src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1529,11 +1529,11 @@ public function getRemovedBindingIds()
     /**
      * Adds a removed binding id.
      *
-     * @param int $id
+     * @param string $id
      *
      * @internal
      */
-    public function addRemovedBindingIds($id)
+    public function addRemovedBindingId($id)
```

Comment:
wouldn't it be even better to name this `addRemovedBindings`?

Reply:
`addRemovedBinding`? Since we don't use the plural form for adder / remover? Or should it be changed to take an array?

Reply:
but this is not a simple adder. It is not adding `$id` in the list. It adds all bindings of this id.

Reply:
the new method is not good either though. It makes me think this id is a binding id (which is a concept we have). But that's not the case.

Reply:
yep, I think `addRemovedBindings` fixes most of these (and I've no better idea :) )

Reply:
What about:

```php
    /**
     * Removes bindings for a service.
     *
     * @param string $id
     *
     * @internal
     */
    public function removeBindings($id)
    {
```

That' what it does, right? If I understand the code correctly...

Reply:
Works for me